### DISCUSSION TITLE
Possible to use Voice

### DISCUSSION THREAD

**Discussion by Freacly** [JurajNyiri/HomeAssistant-Tapo-Control#797] [Discuss]
Hello,

First of all, this Integration is working mostly good with my cameras.
Thank you for that custom integration.

Ive got a question about the microphone usage.

Is it possible to use the microphone function over home Assistent?
Like press the button and talk.

I cant find any solution or entity to realize that.

**Comment by JurajNyiri** [comment]
Yes via go2rtc

**Reply by Freacly** [reply]
How to configure this?
Do you have a link where i can read more about IT?

**Reply by JurajNyiri** [reply]
https://github.com/AlexxIT/go2rtc
